### PR TITLE
docs: add closing tag in hierarchical injectors documentation

### LINKED ISSUE #54938: Docs: missing closing tag in logical structure of the template
### Describe the problem that you experienced

There are missing closing tags on the last code example. 
https://angular.dev/guide/di/hierarchical-dependency-injection#logical-structure-of-the-template

it should be:
<code-example format="html" language="html">
&lt;app-root&gt;
  &lt;#VIEW&gt;
    &lt;app-child&gt;
    &lt;#VIEW&gt;
      &hellip;content goes here&hellip;
    &lt;/#VIEW&gt;
    &lt;/app-child&gt;
  &lt;/#VIEW&gt;
&lt;/app-root&gt;
</code-example>

And I wanted to push my PR but it seems that I don't have permission to the project. Could you help me to how to contribute as well? 

### Enter the URL of the topic with the problem

_No response_

### Describe what you were looking for in the documentation

_No response_

### Describe the actions that led you to experience the problem

_No response_

### Describe what you want to experience that would fix the problem

_No response_

### Add a screenshot if that helps illustrate the problem

_No response_

### If this problem caused an exception or error, please paste it here

_No response_

### If the problem is browser-specific, please specify the device, OS, browser, and version

_No response_

### Provide any additional information here in as much as detail as you can

_No response_

### PR DESCRIPTION
**Pull request by mlz11**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Fixes #54938.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by JeanMeche — CHANGES_REQUESTED** [review]
Hi, can you revert the formating changes you introduced when saving the file ? 
We'd like the change to only be the doc fix.

**Comment by mlz11** [comment]
> Hi, can you revert the formating changes you introduced when saving the file ? We'd like the change to only be the doc fix.

Yes, done ✅ 
I thought it was okay since `yarn ng-dev format changed` and `yarn lint` were not catching anything. I guess they don't scan `.md` files ?

**Review by sonukapoor — APPROVED** [review]
Looks good now 👍

File: adev/src/content/guide/di/hierarchical-dependency-injection.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ The following is an example of how the `<app-root>` and `<app-child>` view trees
      </#VIEW>
     </app-child>
   <#VIEW>
```
Comment (JeanMeche) [inline adev/src/content/guide/di/hierarchical-dependency-injection.md:341]: This one should also be closed.

Comment (mlz11) [reply adev/src/content/guide/di/hierarchical-dependency-injection.md]: Done ✅

**Review by bencodezen — APPROVED** [review]

**Review by JeanMeche — APPROVED** [review]

**Comment by atscott** [comment]
This PR was merged into the repository by commit 6484ac21f99dcaac36e92db7ecfb1545d85c9910.

**Pull request closed without merging**